### PR TITLE
Highlight task

### LINKED ISSUE #32: New plugin: highlight code
Might require splitting the render if half so hiccup can be modified. Or could maybe work on existing html. (#30).

But first we need highlighting lib usable from Boot. There doesn't seem to be any maintained Java implementations. [Jygments](https://github.com/tliron/jygments) looks the most useful but it is not released on maven.

Instead we could use a pod with Jython and use Pygments (a python lib). Example https://github.com/tailrecursion/boot-hoplon/blob/master/src/tailrecursion/boot_hoplon/pygments.clj.

### PR DESCRIPTION
**Pull request by jstaffans**
Add syntax highlighting task (fixes #32). Highlighting is done using Pygments and the [clygments](https://github.com/bfontaine/clygments) wrapper. I adapted a set of [guidelines for adding syntax highlighting to Stasis sites](http://cjohansen.no/building-static-sites-in-clojure-with-stasis/) for this.

Works by modifying HTML output. There are tests for markdown- and asciidoc-produced HTML.

If this gets merged, I'll make a PR to https://github.com/hashobject/perun.io as well to update the documentation of the built-in tasks.

### CODE REVIEW

**Comment by bhagany** [comment]
This is great! I'd like to give some time for other committers to weigh in before merging, but it meets with my approval.

**Review by podviaznikov — APPROVED** [review]
Looks good to me too

**Comment by podviaznikov** [comment]
@jstaffans go ahead and create documentation PR. Thank you for this one!

**Comment by podviaznikov** [comment]
btw, @jstaffans feel free to add your site to the examples section in the readme if you want

**Pull request merged**
